custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -187,6 +187,26 @@ def _get_mapping(field: Fields, values: List[Values]) -> Map:
     return Map({"field": Enum(str(field)), "condition": conditions, "value": value_enum})
 
 
+def check_is_mm(data):
+    if data.shape[1] == 1 and "%%MatrixMarket matrix coordinate" in data.columns[0]:
+        return True
+    else:
+        return False
+
+
+def check_csr_sparse(data):
+    try:
+        return data.format == "csr"
+    except AttributeError:
+        return False
+
+
+def check_is_sparse(data):
+    # this is more verbose than ideal since or doesn't seem to correctly short circuit with the data series apply,
+    # causing an issue dtypes missing in this case.
```

Comment:
this comment is hard to understand

Reply:
old comment that is no longer relevant, removing